src/schema/me/create_bidder_mutation.js

```diff
@@ -0,0 +1,30 @@
+import { GraphQLNonNull, GraphQLString } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import Bidder from "schema/bidder"
+
+export default mutationWithClientMutationId({
+  name: "Bidder",
+  description: "Create a bidder",
+  inputFields: {
+    sale_id: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
+  },
+  outputFields: {
+    bidder: {
+      type: Bidder.type,
+      resolve: bidder => bidder,
+    },
+  },
+  mutateAndGetPayload: (
+    { sale_id },
+    request,
+    { rootValue: { accessToken, userID, createBidderLoader } }
+  ) => {
+    if (!accessToken) {
+      return new Error("You need to be signed in to perform this action")
+    }
+
+    return createBidderLoader({ user_id: userID, sale_id })
```

Comment:
I don't think you need to pass in `user_id` (which only works for admins registering a different user), this endpoint uses `current_user` so you only need to pass in `sale_id`.

Reply:
Oh good point! Especially since this is nested under `me`.